src/ui/components/confirm.c

```diff
@@ -20,6 +20,7 @@
 
 #include <hardfault.h>
 #include <screen.h>
+#include <ui/fonts/arial_fonts.h>
```

Comment:
Why is this include needed only now. What happens if you don't include?

Reply:
oops that's a leftover from when i used a font inside _render(). will remove again